list_option.go

```diff
@@ -178,6 +178,41 @@ func (l ListFollowersOption) addQuery(req *http.Request) {
 	}
 }
 
+type ListSpecifiedUserMemberOfOption struct {
+	Expansions      []Expansion
+	ListFields      []ListField
+	MaxResults      int
+	PaginationToken string
+	UserFields      []UserField
+}
+
+func (l *ListSpecifiedUserMemberOfOption) addQuery(req *http.Request) {
+	q := req.URL.Query()
+	if len(l.Expansions) > 0 {
+		for _, expansion := range l.Expansions {
+			if expansion != "list.owner_id" {
+				panic("lists a specified user is a member of: Only owner_id is allowed")
+			}
```

Comment:
It's not required, now.
And, if you do it, don't use panic.
It may occur unexpected behavior for someone who will use `gotwtr`.